front/src/modules/settings/data-model/components/SettingsObjectFieldTypeSelectSection.tsx

```diff
@@ -88,7 +88,7 @@ export const SettingsObjectFieldTypeSelectSection = ({
           FieldMetadataType.Boolean,
           FieldMetadataType.Currency,
           FieldMetadataType.DateTime,
-          FieldMetadataType.Enum,
+          FieldMetadataType.Select,
```

Comment:
you are the 3rd person fixing this :p You might have conflict with main, you'll need to rebase or merge main into your branch!